app/controllers/util/FormField.scala

```diff
@@ -210,14 +212,12 @@ object TextField {
 
   def validate(text: String, textField: TextField): Option[String] =
     if (text == "") {
-      val fieldType: String = inputModeToFieldType(textField.inputMode)
-      val fieldName = if (textField.fieldId == "TitleAlternate") "alternative title" else textField.fieldName
-      Some(emptyValueError.format(fieldType, fieldName))
+      val fieldName = if (textField.fieldAlternativeName.isEmpty) textField.fieldName.toLowerCase else textField.fieldAlternativeName.toLowerCase
```

Comment:
Sorry, can we move this logic to `FormField`?

Maybe something like this:

```
abstract class FormField {
  val fieldId: String
  val fieldName: String
  val fieldAlternativeName: String
  val fieldDescription: String
  val multiValue: Boolean
  val isRequired: Boolean
  val hideInputs: Boolean = false
  val fieldErrors: List[String]
  val dependencies: Map[String, List[FormField]] = Map.empty
  def selectedOptionNames(): List[String]

  def getAlternativeName: String = {
    if (fieldAlternativeName.nonEmpty) fieldAlternativeName else fieldName
  }
}
```

We can use this new method to get the AlternativeName in all the `validate` methods.

Reply:
This sounds like a reasonable idea, since all fields can have an alternative name. I've made the changes as suggested